### ISSUE TITLE
Final training loss is 0.0000 and training loss while training too small, eg: loss=5.87e-7

### ISSUE THREAD

**Issue by medoidai** [benfred/implicit#335]
Hi I would like to ask how you would interpret the fact that after using the ALS algorithm with some weighted preference matrix I get final training loss 0.0000 and various losses while training with calculate_training_loss=True which are very close to 0, eg: 5.87e-7. Can I think of it as an overfitting? Thank you.

2020-03-20 13:56:36,661 - implicit - DEBUG - Running 20 ALS iterations
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [02:32<00:00,  7.63s/it, loss=5.78e-7]
2020-03-20 13:59:09,333 - implicit - INFO - Final training loss 0.0000

**Comment by medoidai** [comment]
Problem solved guys.
